app/controllers/users_controller.rb

```diff
@@ -1,35 +1,90 @@
+
 class UsersController < ApplicationController
-  before_action :load_user, only: [:show, :edit]
-  
-  def new; end
+  before_action :logged_in_user, except: %i(show new create)
+  before_action :load_user, only: %i(edit update destroy)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: destroy
+
+  def index
+    @users = User.paginate(page: params[:page])
+  end
 
-  def new
+  def show
+    @user = User.find_by params[:id]
```

Comment:
shoud define load_user function to find user
```
before_action :load_user, only: [:edit, :update, :destroy, :show]
def load_user
    @user = User.find_by params[:id]
end
```
and call it by using before_action

Reply:
not fix yet

Reply:
i try to fix this but im not sure

Reply:
remove this line because  you have defined above
  before_action :load_user, only: [:edit, :update, :destroy, :show]

Reply:
Done